UnityAds/UnityAdsInterstitialCustomEvent+UnityAdsLoadDelegateAdditions.h

```diff
@@ -0,0 +1,36 @@
+//
+//  UnityAdsInterstitial+UnityAdsLoadDelegateAdditions.h
+//  MoPubSDK
+//
+//  Copyright © 2021 MoPub. All rights reserved.
+//
+
+#ifndef UnityAdsInterstitial_UnityAdsLoadDelegateAdditions_h
+#define UnityAdsInterstitial_UnityAdsLoadDelegateAdditions_h
+
+#import <UnityAds/UnityAds.h>
+#import "UnityAdsInterstitialCustomEvent.h"
+
+/**
+ *  The `UnityAdsLoadDelegate` protocol defines the required methods for receiving messages from UnityAds.load() method.
+ */
+@interface UnityAdsInterstitialCustomEvent (UnityAdsLoadDelegate)
```

Comment:
It might be useful to create an abstract class for the Event. I assume that Interstitial and Rewarded event will reuse the same flow of load/show calls to UnityAds. It might help us to reuse the logic.

Reply:
Yeah, I was focusing on just getting interstitial working and cleaned up.  Then I could move forward to see if/how things could be extracted to let rewarded share much of the code.